nomenclature/testing.py

```diff
@@ -28,9 +29,13 @@ def assert_valid_yaml(path: Path):
 
 
 def assert_valid_structure(
-    path: Path, mappings: str = "mappings", definitions: str = "definitions"
+    path: Path,
+    mappings: str = "mappings",
+    definitions: str = "definitions",
+    dimensions: List[str] = ["region", "variable"],
```

Comment:
Since you added a new parameter, it would need an entry in the doc string below.